src/DistortableCollection.js

```diff
@@ -15,6 +15,8 @@ L.DistortableCollection = L.FeatureGroup.extend({
     L.Utils.initTranslation.call(this);
 
     this.editable = this.options.editable;
+    this.tooltipControl = options.tooltipControl;
```

Comment:
I think this is the testing issue - should it be this.options or do we need to catch undefineds?

Reply:
I resolved the testing issue, no problems. Many thanks!